language/go/testdata/platforms/BUILD.want

```diff
@@ -21,9 +21,15 @@ go_library(
     _gazelle_imports = [
         "example.com/repo/platforms/generic",
     ] + select({
+        "@io_bazel_rules_go//go/platform:android": [
```

Comment:
Let's also test that a file with the `_android.go` suffix and no `+build` tag gets included.

I think that makes sense in this test case, but it could be anywhere in the generator test.

Reply:
done!